redfish-core/lib/account_service.hpp

```diff
@@ -1541,6 +1541,138 @@ inline void updateUserProperties(
     });
 }
 
+inline void uploadACF(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                      const std::vector<uint8_t>& decodedAcf)
+{
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    sdbusplus::message::message& m,
+                    const std::tuple<std::vector<uint8_t>, bool, std::string>&
+                        messageFDbus) {
+        if (ec)
+        {
+            BMCWEB_LOG_ERROR("DBUS response error: {}", ec);
```

Comment:
In bmcweb, it is suggested to use `ec.value()` in log.

Reply:
okay

Reply:
In the past, there have been mixed uses of ec in logging.
Now, it has been suggested to use `ec.value()` in logging -- which will reduce the amount of logging output.